reputation/microeconomic_reputation.tex

```diff
@@ -0,0 +1,367 @@
+
+\item Every agent can costlessy access and analyze all of the publicly available information about Golem. Thanks to this assumption we can remove $I$ from the equations - we no longer
+    care about "what agent knows", but only about "what really is". This is a major simplification that might not be a good approximation of the "real world" Golem market, but we
+    accept it for the sake of the brevity of this document. Also "truth about Golem is known" is an ideal world we would like to live in.
+
+\item The expected utility and money are interchangeable, that is - for every agent $A$ and a pair of decisions $(D_1, D_2): E_A(D_1) < E_A(D_2)$, there is an amount of money X that
+    can be given to the agent so that $E_A(D_1) + U_A(X) = E_A(D_2)$. This way we can treat utilities as if they were money, and thus compare them between different agents.
+
+\item Lets define the Golem Value as:
+
+\begin{equation}
+    V_G = \sum_{A \in all\_agents\_using\_Golem}E_A(USE\_GOLEM)
+\end{equation}
+(Note that - because of assumption 1 - $E_A(USE\_GOLEM)$ is always positive, decision "use Golem" is only made by agents who expect to profit from it).
+
+We assume that maximization of $V_G$ is one of the goals of the Golem Factory - and from the point of view of this document, the only goal.
+\end{enumerate}
+
+\section{The "reputation" and its purpose}
+\subsection{Notation}
+
+We'll be using a bunch of different symbols, but they follow a clear common pattern, so please don't be frightened.
+
+\begin{itemize}
+\item $V_A(a)$ is the total value of the agreement $a$ from the POV of agent $A$. As per assumption 3, this is "utility expressed as money".\footnote{
+I.e. $V_A(a) = X$ means "when agent $A$ takes part in the agreement $a$, their hapiness changes as if they were given $X$ money".}, i.e.:
+    \begin{itemize}
+        \item For the provider it is the utility of the money received decreased by the utility cost of the hardware/electricity/etc.
+        \item For the requestor it is the utility of the resources obtained decreased by the utility cost of the money spent
+    \end{itemize}
+\item $V_P(a)$/$V_R(a)$ are agreement values from the POV of (respectively) provider/requestor.
+\item $V_{PN}(a)$/$V_{RN}(a)$ are nominal (i.e. negotiated) values of the agreement\footnote{
+   Note that this doesn't translate directly to the agreement amount. E.g. when agreed amount is positive but so low it doesn't cover provider's costs the $V_{PN}$ is negative.
+}. They equal $V_P(a)$/$V_R(a)$ if neither side breached the agreement $a$.
+\item $V_{AL}(a)$ is the value lost by agent $A$ because of the other side breaching the agreement $a$.
+\item $V_{AG}(a)$ is the value gained by agent $A$ when they break the agreement $a$.
+\item $V_{PL}$, $V_{RL}$, $V_{PG}$, $V_{RG}$ are $V_{AL}$/$V_{AG}$ from the POV of provider/requestor.
```

Comment:
In which cases is `V_PL != V_RG`? I realise you offer some examples below, but a more refined analysis might be worthwhile.

Reply:
Added a minor comment in ac044d4.

This might need some bigger refactoring (e.g. so that there won't be a forward reference to a footnote), but I'll wait with this until we have agreement about the contents.

Reply:
I agree that it needs more love, especially that your whole reasoning crucially depends on `V_L - V_G` being positive. However, it might be worth considering what really happens if we drop this assumption and pursue the avenue I suggest in  https://github.com/golemfactory/golem-architecture/pull/45#discussion_r804606727 (value of information) instead.

Reply:
The avenue might be interesting, I'll think about this.